src/CMakeLists.txt

```diff
@@ -149,6 +149,9 @@ if(IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/jit32")
 endif()
 
 add_subdirectory(gc)
```

Comment:
Yep, we can certainly do that! I can't think of any reason why that couldn't be done today.